loops-arrays-objects.js

```diff
@@ -48,22 +60,42 @@ const scores = [
 //  { name: 'Rianne', score: 66, grade: 'D' }
 //  ];
 // ==========================================
-
-
-
-
+for (let i = 0; i < scores.length; i++) {
+    if(scores[i].score < 60) {
+        scores[i].grade = "F";
+    }
+    else if (scores[i].score < 70) {
+        scores[i].grade = "D";
+    }
+    else if (scores[i].score < 80) {
+        scores[i].grade = "C";
+    }
+    else if (scores[i].score < 90) {
+        scores[i].grade = "B";
+    }
+    else if (scores[i].score < 100) {
+        scores[i].grade = "A";
+    }
+}
+
+console.log(scores);
```

Comment:
hier weer precies hetzelfde als bij de vorige opdracht.  Je hebt vaak erg veel regels code in een opdracht. Het zo kort mogelijk maken is hierbij wel handig . Verder weer perfect gedaan

Reply:
Top ja inderdaad!